docs/source/tutorial.rst

```diff
@@ -220,6 +220,10 @@ Let's mark the document as "private":
 .. include:: tutorial/mark-bid-doc-private.http
    :code:
 
+`Public` end-point shouldn't provide getting information about private document. Use primary server to download private document. For example :
+`https://lb.api-sandbox.openprocurement.org/api/2.3/tenders/019866978644451cb141683276efb4d7/bids/4343419b8ea840a69dc090e473801cc8/documents/d786d1697c0541b2b908111f56aae23e?download=35a83b81153f4b6f8c8be79f19114433`
+Before that, as before getting any private document, we need to use`authentication<http://documentservice.api-docs.openprocurement.org/en/latest/authentication.html>`_  and authorization (to get a token).
```

Comment:
to use `authentication <http://documentservice.api-docs.openprocurement.org/en/latest/authentication.html>`_ and